src/rp2/plugin/accounting_method/lifo.py

```diff
@@ -15,25 +15,26 @@
 from typing import Optional
 
 from rp2.abstract_accounting_method import (
-    AbstractAccountingMethod,
+    AbstractAcquiredLotCandidates,
+    AbstractHeapAccountingMethod,
     AcquiredLotAndAmount,
-    AcquiredLotCandidates,
     AcquiredLotCandidatesOrder,
+    AcquiredLotHeapSortKey,
 )
 from rp2.abstract_transaction import AbstractTransaction
 from rp2.in_transaction import InTransaction
 from rp2.rp2_decimal import ZERO, RP2Decimal
 
 
-# LIFO plugin. See https://www.investopedia.com/terms/l/lifo.asp. This plugin uses universal application, not per-wallet application:
+# Last In, First Out (LIFO) plugin. See https://www.investopedia.com/terms/l/lifo.asp. This plugin uses universal application, not per-wallet application:
 # this means there is one queue for each coin across every wallet and exchange and the accounting method is applied to each such queue.
 # More on this at https://www.forbes.com/sites/shehanchandrasekera/2020/09/17/what-crypto-taxpayers-need-to-know-about-fifo-lifo-hifo-specific-id/
 # Note that under LIFO the date acquired must still be before or on the date sold: for details see
 # https://ttlc.intuit.com/community/investments-and-rental-properties/discussion/using-lifo-method-for-cryptocurrency-or-even-stock-cost-basis/00/1433542
-class AccountingMethod(AbstractAccountingMethod):
+class AccountingMethod(AbstractHeapAccountingMethod):
```

Comment:
Change the comment at line 44 in positive terms, similar to what we did for FIFO (i.e.: "This loop features...").

Reply:
Done in newest commits